query/src/main/scala/filodb/query/exec/rangefn/AggrOverTimeFunctions.scala

```diff
@@ -272,8 +272,13 @@ class MedianAbsoluteDeviationOverTimeFunction(funcParams: Seq[Any]) extends Rang
 }
 
 class LastOverTimeIsMadOutlierFunction(funcParams: Seq[Any]) extends RangeFunction {
-  require(funcParams.size == 1, "last_over_time_is_mad_outlier function needs a single tolerance argument")
-  require(funcParams.head.isInstanceOf[StaticFuncArgs], "tolerance parameter must be a number")
+  require(funcParams.size == 2, "last_over_time_is_mad_outlier function needs a two scalar arguments" +
+                  " (tolerance, bounds)")
+  require(funcParams(0).isInstanceOf[StaticFuncArgs], "first tolerance parameter must be a number")
+  require(funcParams(1).isInstanceOf[StaticFuncArgs], "second bounds parameter must be a number(0, 1, 2)")
```

Comment:
nitpick: can we add some documentation on why this must be between 0 - 2 and what is the role of tolerance and bound parameter ?